### PR TITLE
Generate example with factory

### PR DESCRIPTION
**Pull request by fernandopiovezan1**
Firstly, thank you very much for the great tool you created and made available.
Due to the necessity of my project, I need the examples to be generated exactly as the factories provide me, this way I avoid using false examples to consume my documentation.
The possibility of disabling this process has been added for those who do not wish to use this tool.
I didn't do the tests to cover it because I don't know exactly what the flow of this test looks like, if you could explain it to me I'll be happy to adjust the tests to cover this scenario. I have some other changes in the queue that I would like to make, so if it is of interest I will send more PR's to analyze the feasibility of implementation and if they do not find it viable, they can discard the PR.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
         }
     ],
     "require": {
-        "php": "^7.4|^8.0|^8.1|^8.2|^8.3",
+        "php": "^8.0|^8.1|^8.2|^8.3",
```
Comment (kevincobain2000) [inline composer.json:19]: Did it throw error on 7.4? You should keep it.

File: resources/dist/index.html [hunk 2]
```diff
@@ -1 +1 @@
...hild.nodeValue&&(this.lastChild.remove(),t())}));e.observe(this,{childList:!0}),document.addEventListener("DOMContentLoaded",t)}else this.childrenConnectedCallback()}async childrenConnectedCallback(){let t=this.getAttribute("before-hydration-url");t&&await import(t),this.start()}start(){let t=JSON.parse(this.getAttribute("opts")),e=this.getAttribute("client");void 0!==Astro[e]?Astro[e]((async()=>{let t=this.getAttribute("renderer-url"),[e,{default:n}]=await Promise.all([import(this.getAttribute("component-url")),t?import(t):()=>()=>{}]),r=this.getAttribute("component-export")||"default";if(r.includes(".")){this.Component=e;for(let t of r.split("."))this.Component=this.Component[t]}else this.Component=e[r];return this.hydrator=n,this.hydrate}),t,this):window.addEventListener(`astro:${e}`,(()=>this.start()),{once:!0})}attributeChangedCallback(){this.hydrate()}},n(t,"observedAttributes",["props"]),t))}})()</script><astro-island client=only component-export=default component-url=/request-docs/_astro/App.D5ip0zGO.js opts={&quot;name&quot;:&quot;App&quot;,&quot;value&quot;:true} props={} renderer-url=/request-docs/_astro/client.D9Vng9vH.js ssr="" uid=Z26kMUL></astro-island></body></html>
```
Comment (kevincobain2000) [inline resources/dist/index.html:1]: Sorry, you didn't need to push this. This is handled by our CI. You can remove it in the next commit. Same thing for all `resrouces/dist`

**Comment by kevincobain2000** [comment]
Interesting. Thanks.
I am ok with your choice of implementation.
Can you push another commit to address the comments and update `README.md` explaining the usage too?

Thanks, and I ll merge it within today.

**Review by kevincobain2000 — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (fernando-piovezan-instar) [reply composer.json:19]: Sorry for not explaining
In the file [LaravelRequestDocsToOpenApi](https://github.com/rakutentech/laravel-request-docs/blob/dc003526eba0fe747197c309c2adbc6fd7308ed6/src/LaravelRequestDocsToOpenApi.php#L111), there is a mention of str_contains which was only added in PHP 8+ for this reason I removed the reference to PHP 7.4 from composer
Reference [str_contains](https://www.php.net/manual/en/function.str-contains.php)

**Comment by kevincobain2000** [comment]
Sorry haven't pushed the new tag yet, as UT failed.
https://github.com/rakutentech/laravel-request-docs/actions/runs/9172478571/job/25219068517
I'd need to revert this commit

**Comment by kevincobain2000** [comment]
Can you re-open this PR with UT fixed?
Thanks

**Pull request merged**